core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -1086,4 +1086,40 @@ default TimeType getTimeType(String tableName) throws TableNotFoundException {
     throw new UnsupportedOperationException();
   }
 
+  /**
```

Comment:
Is there a good place to summarize what ondemands means vs on or offline?  I assume the web site would have a more lengthy description, but having something in the javadocs might - would not want to see it on every ondemand method,  but somewhere? (and can be done later)